src/test/java/LoginPageTest.java

```diff
@@ -0,0 +1,31 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.LoginPage;
+import util.Constants;
+
+
+public class LoginPageTest {
+    WebDriver driver;
+
+    @BeforeMethod(alwaysRun = true)
+    public void preparingData() {
+        System.setProperty("webdriver.chrome.driver", "C:\\chromedriver_win32\\chromedriver.exe");
```

Comment:
to avoid the issue with webdriver add this dependency to the pom file
```
<dependency>
            <groupId>io.github.bonigarcia</groupId>
            <artifactId>webdrivermanager</artifactId>
            <version>5.2.1</version>
        </dependency>
```
and 
```
WebDriverManager.getInstance(ChromeDriver.class).setup();
driver = new ChromeDriver();
```
and remove 16 line